.github/testing/config.yml

```diff
@@ -0,0 +1,16 @@
+DRYDOCK_INIT_JOBS: true
+K8S_NAMESPACE: testing-openedx
+PLUGINS:
+- forum
+- mfe
+- minio
+- drydock
+DRYDOCK_FLOWER: true
+DRYDOCK_INGRESS: true
+DRYDOCK_DEBUG: true
+DRYDOCK_ENABLE_CELERY_TUNING: true
+DRYDOCK_ENABLE_MULTITENANCY: true
+DRYDOCK_ENABLE_SCORM: true
+DRYDOCK_ENABLE_SENTRY: true
```

Comment:
Can we have another configuration where all the features are turned off to test it?

Reply:
this wouldn't change anything as all resources are created in different files and added via Kustomize. So, the definition exists and it's validated even when the variables are disabled